doc/pages/Installation.md

```diff
@@ -178,11 +177,13 @@ cmake -DENABLE_PYTHON=ON ..
 
 + Quite often there are multiple Python versions installed in a system. This is likely the cause of many (if not most) of the installation problems related to Python. To prevent conflicts among them, one can explicitly refer to the Python version to be used. Example:
   ```
-  -DCMAKE_PYTHON_EXECUTABLE=/usr/bin/python
-  -DCMAKE_PYTHON_INCLUDE_DIR=<path_to_folder_containing_Python.h>
-  -DCMAKE_PYTHON_LIBRARY=<path_to_file>/libpython<version_tag>.so
+  -DPython_EXECUTABLE=/usr/bin/python
+  -DPython_INCLUDE_DIRS=<path_to_folder_containing_Python.h>
+  -DPython_LIBRARY=<path_to_file>/libpython<version_tag>.so
   ```
   Note that in systems running OSX, the extension .so should be replaced by .dylib.

```

Comment:
Should we also add something like the following lines?

"The path to the location wehre the crpropa python module will be installed, can be specified with:
`-DPython_INSTALL_PACKAGE_DIR=`"